community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/spi/CSVResources.scala

```diff
@@ -41,6 +41,8 @@ object CSVResources {
     override def bufferSize(): Int = DEFAULT_BUFFER_SIZE
 
     override def multilineFields(): Boolean = true
+
+    override def emptyQuotedStringsAsNull(): Boolean = false
```

Comment:
should we skip those properties by default? (maybe we can have it = `true` by default instead?)